.ci-dockerfiles/idris-4e70401.dockerfile

```
@@ -0,0 +1,11 @@
+from haskell:8.2
+
+run apt-get update && apt-get install -y libncurses5-dev
+
+run git clone https://github.com/idris-lang/Idris-dev && \
+  cd Idris-dev && \
+  git checkout 4e70401 && \
+  cabal update && \
+  make -j install
```

Try using `-j1` instead of `j`. We had the same problems with Circle CI. It runs out of memory when tries to build with cabal in parallel with multiple threads.

```suggestion
  make -j1 install
```